maggy/ablation/ablator/loco.py

```diff
@@ -213,12 +232,16 @@ def create_trial_dict(self, ablated_feature=None, layer_identifier=None):
             )
             trial_dict["ablated_feature"] = ablated_feature
 
-        if layer_identifier is None:
+        # 2 - determine the model generation logic
+        # 2.1 - no model ablation
+
+        if layer_identifier is None and custom_model_generator is None :
             trial_dict[
                 "model_function"
             ] = self.ablation_study.model.base_model_generator
             trial_dict["ablated_layer"] = "None"
-        else:
+        # 2.2 - layer ablation based on base model generator
+        elif layer_identifier is not None and custom_model_generator is None:
```

Comment:
```suggestion
        elif layer_identifier and not custom_model_generator:
```
same as above, just syntactic sugar